test/SortTest.cpp

```diff
@@ -369,43 +340,87 @@ TEST(Sort, externalSortLazyOutput) {
   // Clear cache at start to avoid hits from previous tests.
   qec->getQueryTreeCache().clearAll();
 
-  // Create input table with different values than other tests.
+  // Create input table large enough to exceed the second threshold of 100 KB
```

Comment:
I notice you're avoiding in definite articles in these types of comments, why not `Create an input table [...]` for example? Any specific reason?